timescaledb/getting-started/compress-data.md

```diff
@@ -60,16 +48,35 @@ compression option and its `orderby` and `segmentby` attributes:
 SELECT * FROM timescaledb_information.compression_settings;
 ```
 
-Now that compression is enabled, schedule a policy to automatically compress
-data according to the settings defined above. Set a policy to compress data
-older than 10 years by using the following query:
+Sample results:
+```
+hypertable_schema|hypertable_name |attname|segmentby_column_index|orderby_column_index|orderby_asc|orderby_nullsfirst|
+-----------------+----------------+-------+----------------------+--------------------+-----------+------------------+
+public           |stocks_real_time|symbol |                     1|                    |           |                  |
+public           |stocks_real_time|time   |                      |                   1|false      |true              |
+```
+
+## Automatic compression
+When you have enabled compression, you can schedule a [policy to automatically compress][compress-automatic]
+data according to the settings defined above. 
+
+For example, if you want to compress data on your hypertable that has existed more than a month
+in the past, you could use this code:
 
 ```sql
--- Add compression policy
-SELECT add_compression_policy('weather_metrics', INTERVAL '10 years');
+SELECT add_compression_policy('stocks_real_time', INTERVAL '2 weeks');
 ```
 
-Just like for automated policies for continuous aggregates, we can view information and statistics about our compression background job in the following two information views:
+Similar to the continuous aggregates policy and retention policies, when you run this SQL, all 
+chunks that contain data that is at least two old are compressed in `stocks_real_time`, 
+and a recurring compression policy is created. 
+
+It is important that you don't try to compress all your data. Although you can insert
+new data into compressed chunks, compressed rows can't be updated or deleted. Therefore,
+it is best to only compress data after it has aged, once data is less likely to require updating.  
+
+Just like for automated policies for continuous aggregates, you can view information and statistics 
+about your compression background job in these two information views:
 
 ```sql
```

Comment:
Suggest splitting these into two code blocks so the comments can be extracted into the regular text. (They really are very hard to read because the color contrast is so low.)